cstore_reader.c

```diff
@@ -66,8 +71,8 @@ static StripeSkipList * SelectedBlockSkipList(StripeSkipList *stripeSkipList,
 											  bool *selectedBlockMask);
 static uint32 StripeSkipListRowCount(StripeSkipList *stripeSkipList);
 static bool * ProjectedColumnMask(uint32 columnCount, List *projectedColumnList);
-static bool * DeserializeBoolArray(StringInfo boolArrayBuffer, uint32 boolArrayLength);
-static Datum * DeserializeDatumArray(StringInfo datumBuffer, bool *existsArray,
+static void DeserializeBoolArray(StringInfo boolArrayBuffer, bool *boolArray, uint32 boolArrayLength);
+static void DeserializeDatumArray(StringInfo datumBuffer, bool *existsArray, Datum *datumArray,
 									 uint32 datumCount, bool datumTypeByValue,
 									 int datumTypeLength, char datumTypeAlign);
```

Comment:
1. First line exceeds the 90 character limit.
2. Second and third lines not aligned with the first line.